.github/workflows/test-scripts.yml

```diff
@@ -1,17 +1,30 @@
-name: test-scripts
+name: upstream-tests
 on:
   pull_request:
     paths:
-    - 'scripts/**'
-    - 'Makefile'
+    - '**'
+    - '!doc/contributors/**'
+    - '!doc/design/**'
   schedule:
   - cron: '0 0 * * *' # daily to pick up releases
 jobs:
+  image-build:
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Check out the repo
+      uses: actions/checkout@v2
+    - name: Push to Docker Hub
```

Comment:
Mind changing this step name since push is disabled?

Reply:
ah, sorry -- will do!